var/spack/repos/builtin/packages/lapackpp/package.py

```diff
@@ -45,9 +45,19 @@ class Lapackpp(CMakePackage):
 
     def cmake_args(self):
         spec = self.spec
+
+        blas_vendor = ''
+        cxx_flags = ' '
+        if (spec['blas'].name == 'cray-libsci'):
+            blas_vendor = '-DBLA_VENDOR=CRAY'
```

Comment:
Instead of adding empty flags, make `args` a local variable and `args.append(self.define('BLA_VENDOR', 'CRAY'))`.